build.gradle

```diff
@@ -56,62 +57,91 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.1.0')
-    api 'com.exactpro.th2:common:3.44.0'
-    api "com.exactpro.sf:sailfish-core:${sailfishVersion}"
+    api platform('com.exactpro.th2:bom:4.5.0')
+    api('com.exactpro.th2:common:5.4.0-dev') {
+        exclude group: 'com.exactpro.th2', module: 'cradle-core'
+        exclude group: 'com.exactpro.th2', module: 'cradle-cassandra'
+    }
+    api "com.exactpro.sf:sailfish-common:${sailfishVersion}"
+
+    implementation('com.squareup.okio:okio:3.5.0') {
+        because('fix vulnerability in transitive dependency <com.squareup.okhttp3:okhttp>')
+    }
+
+    implementation 'com.exactpro.th2:common-utils:2.2.0-dev'
 
     implementation "org.slf4j:slf4j-api"
     implementation "com.google.protobuf:protobuf-java-util"
 
     // Use JUnit Jupiter API for testing.
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.9.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.0'
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:${kotlin_version}"
```

Comment:
We don't need version for kotlin artifacts because the plugin will resolve them automaticaly